repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/filters/apivalidator/jsonschemacheck/validator.cfg.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<validators xmlns='http://openrepose.org/repose/validator/v1.0'>
+    <validator
+            role="group1"
+            default="false"
```

Comment:
Since this is the only defined validator, either `group1` needs to be in the requests `X-Roles` header or this validator needs the default attribute set to `true`.